src/js/tabs/motors.js

```diff
@@ -772,19 +772,8 @@ motors.initialize = async function (callback) {
 
             $('div.digitalIdlePercent').toggle(protocolConfigured && digitalProtocol);
```

Comment:
We could further simplify using 

```suggestion
            $('div.digitalIdlePercent').toggle(protocolConfigured && digitalProtocol && !FC.ADVANCED_TUNING.idleMinRpm && !FC.MOTOR_CONFIG.use_dshot_telemetry);
```

Reply:
good!simplified it, use_dshot_telemetry should be avoid here